README.md

```diff
@@ -1,24 +1,71 @@
-# README
+# Members Only! 🚀 
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> In this project, I build a site similar to a private Eventbrite with authentication and association between components. 
 
-Things you may want to cover:
+## Events
+![screenshot](img/event.png)
 
-* Ruby version
+## Users list 
+![screenshot](img/users.png)
 
-* System dependencies
+## Profile
+![screenshot](img/profile.png)
 
-* Configuration
+## Getting Started
+- Go to (https://github.com/Alexoid1/Private-events) and have a look around.
+- Open your Terminal.
+- Install Imagemagick (brew install imagemagick)
+- Clone repo
+- Go to the project's root
+- Type (bundle install)
+- Start server (rails server).
+- In your browser open: http://localhost:3000/
 
```

Comment:
## More elaborate instructions
- [ ] Add instructions on how to run your tests.
- [ ] Add a live link to where the site is live (hosted) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)). Thank you.